content/post/78-ipfs-2019-roadmap.md

```diff
@@ -0,0 +1,31 @@
+
+When taking on Package Managers as a top use case and priority, we realized there was still much we didn’t know about the needs and requirements to support this community - so learning more was our first step. We spun up a new [Package Managers Working Group](https://github.com/ipfs/package-managers) in early Q1, which has been focused on research, knowledge-sharing, and stress-testing the protocol with demos and experiments to target our efforts for the rest of the year. The team has already identified a bunch of low hanging fruit around usability and performance. Want to get involved? [Check out this list of known issues and feature requests!](https://github.com/ipfs/package-managers/blob/master/blockers.md) _(and add more!)_ 💪
+
+When we started out with 2019 planning, we actually took on **THREE** top-level priorities instead of one - so you may notice that [working group roadmaps](https://github.com/ipfs/roadmap#2019-working-groups-roadmaps) are a little out of date. It became clear shortly into Q1 that _three_ goals were far too many to focus on. There was lots of overlap between the needs of package managers and large files, but where the goals didn’t overlap it was splitting our focus and resources. Our “decentralized web” goal was also a year (or more) of work in itself - and largely dependent on all the underlying work we were doing to support package managers. Our initial roadmap was too optimistic to achieve in a single year! 🧐
+
+Given this realization, we made the hard choice to descope those goals for 2019 and focus our energies on supporting package managers. However, we’d already finalized our Q1 OKRs and much of the prioritized work was still aligned and valuable for our narrowed priority. Therefore, we decided to pause any additional re-planning until the end-of-quarter mark. Not to fear, Q2 brings lots of fresh package-manager-focused improvements built on all of our great Q1 learnings. Curious what the working groups are picking up and how you can help out? Check out the [IPFS Q2 OKRs](https://docs.google.com/spreadsheets/d/1YSeyWqXh3ImanRrTkYQHHkCofiORn68bYqM_KTLBlsA/edit) and associated planning issues [here](https://github.com/ipfs/team-mgmt/issues/902). 🤗
+
```

Comment:
I updated the index in the Readme tab of https://docs.google.com/spreadsheets/d/1YSeyWqXh3ImanRrTkYQHHkCofiORn68bYqM_KTLBlsA/edit#gid=1681757723 to include package managers